src/PageLayout/PageLayout.tsx

```diff
@@ -178,18 +179,22 @@ const VerticalDivider: React.FC<DividerProps> = ({variant = 'none', variantWhenN
 export type PageLayoutHeaderProps = {
   divider?: 'none' | 'line'
```

Comment:
Non-blocking nitpick because it's unrelated to this PR: : What if instead of having `divider` and `dividerWhenNarrow`, we just had a divider type that looked like this:
```
'none' | 'line' | {
  narrow: 'inherit' | 'none' | 'line' | 'filled' 
  regular: 'none' | 'line'
}
```

Reply:
Totally agree! That's what I'm planning to do next:

![CleanShot 2022-07-26 at 08 33 19@2x](https://user-images.githubusercontent.com/4608155/181048460-03513df0-2b8b-4298-89c6-ac240c146a28.png)